content/applications/sales/crm/track_leads/prospect_visits.rst

```diff
@@ -1,55 +1,116 @@
+
+.. image:: prospect_visits/link-tracker-stats.png
    :align: center
+   :alt: Link tracker statistics page.
 
-See visited pages in your leads/opportunities
-=============================================
+Track page views for leads/opportunities
+========================================
 
-Now each time a lead is created from the contact form it will keep track
-of the pages visited by that visitor. You have two ways to see those
-pages, on the top right corner of your lead/opportunity you can see a
-*Page Views* button but also further down you will see them in the
-chatter.
+When a lead is created from a prospective customer filling out a :ref:`contact form
+<crm/track-leads/webpage-visits/contact-form>` on the website, Odoo tracks every webpage visited by
+that contact. This information is available on the lead/opportunity form in the *CRM* app.
 
-Both will update if the viewers comes back to your website and visits
-more pages.
+To see a prospective customer's page views, navigate to their lead/opportunity form in the
+:guilabel:`CRM` app. The :guilabel:`Page Views` smart button at the top of the form shows the total
+number of times that they viewed a page on the website (including multiple views of the same
+webpage). Click on the smart button to see an organized list of each individual page view.
 
-.. image:: prospect_visits/prospect_visits04.png
+.. image:: prospect_visits/page-views-smart-button.png
    :align: center
+   :alt: Page views smart button on an opportunity form.
 
-.. image:: prospect_visits/prospect_visits05.png
+If a prospective customer clicks on a tracked link, and then fills out a contact form on the website
+creating a lead in Odoo, then the campaign, medium, and source information from the :ref:`link
+tracker <crm/track-leads/link-tracker>` will appear on the lead form. This information appears in
+the :guilabel:`Marketing` section of the :guilabel:`Extra Information` tab of the lead or
+opportunity form.
+
+.. image:: prospect_visits/campaign-medium-source.png
    :align: center
+   :alt: Campaign, medium, and source on an opportunity form.
+
+.. _crm/track-leads/webpage-visits/contact-form:
+
+Configure a contact form
+------------------------
+
+To configure a website contact form to create a lead or opportunity in the *CRM* app when it is
+filled out, navigate to the contact form page on the :guilabel:`Website` app. Then, click the
+:guilabel:`Edit` button at the top right of the page to open the website editor.
+
+Next, click on the form block. In the :guilabel:`Customize` tab of the website editor, in the
+:guilabel:`Form` section, set the :guilabel:`Action` to :guilabel:`Create an Opportunity`.
+
+This creates a lead in the :guilabel:`Leads` section of the *CRM* app (if the :guilabel:`Leads`
+feature is activated in the CRM settings) or an opportunity in the first stage of the pipeline (if
+the :guilabel:`Leads` feature is activated in the CRM settings) when the contact form is filled out.
```

Comment:
```suggestion
Doing so creates a lead in the :guilabel:`Leads` section of the *CRM* app (if the :guilabel:`Leads`
feature is activated in the CRM settings), or an opportunity in the first stage of the pipeline (if
the :guilabel:`Leads` feature is **not** activated in the CRM settings) when the contact form is
filled out.
```
it *would* create an opportunity in the first stage of the pipeline if the Leads feature is NOT activated, right? Or do I have that mistaken?

Reply:
Oh yes that's what I meant to write haha, good catch!